test/no-nested-ternary.js

```diff
@@ -0,0 +1,58 @@
+import test from 'ava';
+import avaRuleTester from 'eslint-ava-rule-tester';
+import rule from '../rules/no-nested-ternary';
+
+const ruleTester = avaRuleTester(test, {
+	env: {
+		es6: true
+	}
+});
+
+const errors = [{ruleId: 'my-rule'}];
```

Comment:
Huh? What is `my-rule`?

---

@MrHen Seems like the `ruleId` property is not correctly checked in the tests...

Reply:
https://github.com/sindresorhus/eslint-plugin-unicorn/issues/368